core/src/main/java/com/netflix/conductor/utils/DateTimeUtils.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 Netflix, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.netflix.conductor.utils;
```

Comment:
There is already a Utils package [here](https://github.com/Netflix/conductor/tree/main/core/src/main/java/com/netflix/conductor/core/utils). Can you move this class under the same package?

Reply:
done.